package.json

```diff
@@ -114,7 +114,7 @@
         "classnames": "^2.2.6",
         "lodash": "^4.17.15",
         "moment": "^2.24.0",
-        "office-ui-fabric-react": "6.189.4",
+        "office-ui-fabric-react": "6.199.0",
```

Comment:
https://github.com/OfficeDev/office-ui-fabric-react/blob/6.0/packages/office-ui-fabric-react/CHANGELOG.md suggests that they're up to 6.200.1 now; should we be going to that?

Reply:
Updated